components/avatar/avatar_presence.story.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
Either just show one instance of the avatar here or remove the controls from showing up. Doesn't make much sense to change the properties when there's multiple preset variants on one page.

Reply:
I think it makes sense to keep the controls for this, so I guess you're saying we should remove this variants file?

Reply:
Either put each instance in a different story, or have all variants displayed without the controls